Product/SoapUI_Test/RegressionSuite/Standard/EntityDocQuery/EntityDocQuery-soapui-project.xml

```diff
@@ -1173,29 +1101,16 @@ count(//ns6:AdhocQueryResponse[1]/ns3:RegistryObjectList[1]/ns3:ExtrinsicObject)
       <con:tearDownScript>nhinc.FileUtils.restoreConfiguration(context.findProperty("GatewayPropDir"), log);</con:tearDownScript>
     </con:testCase>
     <con:testCase failOnError="true" failTestCaseOnErrors="true" id="f15669b3-5b74-4be0-84df-ebec541cbbeb" keepSession="false" maxResults="0" name="Document Query State Fanout" searchProperties="true">
-      <con:settings/>
+      <con:description>This test do a Documet query fan-out based on region US_NY define in NhinTargetCommunities.</con:description><con:settings/>
```

Comment:
There's a typo with "Documet".  Is that worth a new commit now?  If not, it can wait for later.